tests/ttnn/unit_tests/operations/test_moreh_linear.py

```diff
@@ -92,6 +92,8 @@ def moreh_linear(shapes, has_bias, has_output, compute_kernel_config, device):
 @pytest.mark.parametrize("has_bias", [False, True])
 @pytest.mark.parametrize("compute_kernel_options", compute_kernel_options, ids=compute_kernel_ids)
 def test_moreh_linear(shapes, has_bias, compute_kernel_options, device):
+    if has_bias and compute_kernel_options:
+        pytest.skip()
```

Comment:
Why was this skip added?

Reply:
I have added a comment before pytest.skip() in https://github.com/tenstorrent/tt-metal/pull/12812/commits/6f1c3e704ee981bfe35fcf2a4de7739762616e2e to explain. Basically, there is a bug with preserve_fp32_precision with some ops, especially moreh_matmul. This [PR](https://github.com/tenstorrent/tt-metal/pull/12794) fixed it, I have rebased this PR in a local branch, and it fixes the failed test. Now I will have to wait until the PR is merge then I will apply the fix branch.